From 0a427cc638ada13a703b044f38f4b01628c4e620 Mon Sep 17 00:00:00 2001
From: Hyunchul Lee <hyc.lee@gmail.com>
Date: Tue, 13 Jul 2021 15:38:30 +0900
Subject: [PATCH] ksmbd: fix an error message in ksmbd_conn_trasnport_init
Git-commit: 0a427cc638ada13a703b044f38f4b01628c4e620
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Fix an error message in ksmbd_conn_transport_init().

Signed-off-by: Hyunchul Lee <hyc.lee@gmail.com>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/ksmbd/connection.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/ksmbd/connection.c b/fs/ksmbd/connection.c
index 8430848bea45..d7ee0bfb5838 100644
--- a/fs/ksmbd/connection.c
+++ b/fs/ksmbd/connection.c
@@ -372,7 +372,7 @@ int ksmbd_conn_transport_init(void)
 
 	ret = ksmbd_rdma_init();
 	if (ret) {
-		pr_err("Failed to init KSMBD subsystem: %d\n", ret);
+		pr_err("Failed to init RDMA subsystem: %d\n", ret);
 		goto out;
 	}
 out:
-- 
2.33.0

